localize.js

```diff
@@ -126,30 +134,40 @@ const strategies = {
 };
 
 const psuedoLocalizeString = (string, options = { strategy: "accented" }) => {
```

Comment:
There will be conflicts on this line after rebase, I think the best fix is
```js
{ strategy = 'accented', elongateNumbers = false } = {}
```
to keep deep defaults and avoid `can not find elongateNumbers of undefined` Errors in edge cases!